### DISCUSSION TITLE
Enhancing xDrip Statistics

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2921] [Ideas]
This is an idea.

Can we add a "Settings" option to the 3-dot menu on the statistics pages?
What we have now looks like the following.

![Screenshot_20230703-073152](https://github.com/NightscoutFoundation/xDrip/assets/51497406/e36db5e1-579e-445c-8999-7074b5973cf4)

![Screenshot_20230703-073428](https://github.com/NightscoutFoundation/xDrip/assets/51497406/94bd0cc8-f9b8-4e64-bb2d-84b00d683dc0)

If we add a settings option that would open a new page that would have boxes to tick to select what items to add or remove from the page, it would provide some flexibility.

There is an open PR to add the capture rate to the statistics page:  
https://github.com/NightscoutFoundation/xDrip/pull/2698
But, someone may not want to always have the capture rate included in their statistics.
Or, someone may want to add very high and very low to their statistics page.

Would this be acceptable?

**Comment by jstevensog** [comment]
I think this is a good idea.  I cannot think of a situation that I would
use such a thing, but flexibility is always good.
I am sure others would find this very useful.
Cheers

On Mon, 3 Jul 2023 at 21:38, Navid ***@***.***> wrote:

> This is an idea.
>
> Can we add a "Settings" option to the 3-dot menu on the statistics pages?
> What we have now looks like the following.
>
> [image: Screenshot_20230703-073152]
> <https://user-images.githubusercontent.com/51497406/250544152-e36db5e1-579e-445c-8999-7074b5973cf4.png>
>
> [image: Screenshot_20230703-073428]
> <https://user-images.githubusercontent.com/51497406/250544688-94bd0cc8-f9b8-4e64-bb2d-84b00d683dc0.png>
>
> If we add a settings option that would open a new page that would have
> boxes to tick to select what items to add or remove from the page, it would
> provide some flexibility.
>
> There is an open PR to add the capture rate to the statistics page:
> #2698 <https://github.com/NightscoutFoundation/xDrip/pull/2698>
> But, someone may not want to always have the capture rate included in
> their statistics.
> Or, someone may wan to add very high and very low to their statistics page.
>
> Would this be acceptable?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NightscoutFoundation/xDrip/discussions/2921>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACCCZ4YZ56UDXD55SYSMV6DXOKVNDANCNFSM6AAAAAAZ4KT4DE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
John Stevens
"You are how you live, not what you have."
